src/components/OpeningHour.jsx

```diff
@@ -35,12 +35,23 @@ const OpeningHour = ({ schedule, showNextOpenOnly = false }) => {
     </div>;
   }
 
+  const NextTransition = () => {
+    if (!nextTransition || showNextOpenOnly && status !== 'closed') {
+      return null;
+    }
+
+    const options = { nextTransitionTime: nextTransition };
+
+    const text = status === 'closed'
+      ? ` - ${_('reopening at {nextTransitionTime}', 'hour panel', options)}`
+      : ` - ${_('until at {nextTransitionTime}', 'hour panel', options)}`;
```

Comment:
typo: `until at` :arrow_right: `until`

And then we should be ok :smiley:

Reply:
my bad, I should take a deep breath before re-reading...